packages/knip/test/plugins/i18next-parser.test.ts

```diff
@@ -0,0 +1,26 @@
+import { test } from 'bun:test';
+import assert from 'node:assert/strict';
+import { main } from '../../src/index.js';
+import { resolve } from '../../src/util/path.js';
+import baseArguments from '../helpers/baseArguments.js';
+import baseCounters from '../helpers/baseCounters.js';
+
+const cwd = resolve('fixtures/plugins/i18next-parser');
+
+test('Find dependencies with the i18next-parser plugin', async () => {
+  const { issues, counters } = await main({
+    ...baseArguments,
+    cwd,
+  });
+
+  assert(issues.binaries['package.json']['i18next']);
+  assert(issues.devDependencies['package.json']['i18next-parser']);
```

Comment:
This is fine, I'd merge it like this, but what fixtures/tests of others plugins sometimes have is something like this: https://github.com/webpro-nl/knip/blob/main/packages/knip/fixtures/plugins/eleventy3/node_modules/@11ty/eleventy/package.json i.e. a minimal `node_modules` folder with an actual `package.json` to define the `bin` name so that both the binary and the dev dependency will be found (and the `counters` go to zero).